pantheon/src/main/java/tech/pegasys/pantheon/Pantheon.java

```diff
@@ -32,7 +32,7 @@ public static void main(final String... args) {
             new BlockImporter(),
             new RunnerBuilder(),
             new PantheonControllerBuilder(),
-            new Builder());
+            new SynchronizerConfiguration.Builder());
```

Comment:
I admit this change is completely unrelated to this PR but just changes which import we use to make things more readable.